webpack-babel.config.js

```diff
@@ -0,0 +1,20 @@
+var path = require('path');
+
+module.exports = {
+  output: {
+    libraryTarget: 'umd',
```

Comment:
Files from `lib/` will be used like any other NPM module; i.e. passed through Webpack, Browserify, Rollup, or equivalent (or in case there's a very good browser-environment emulation –jsdom?– that can do WebGL, then it could be NodeJS proper), which will rewrite the CommonJS imports.

People using iTowns by other means will use the output from Webpack (in `dist/`, as a UMD module, so compatible with RequireJS/AMD **and** direct use in a `<script src=>`, and also incidentally compatible with CommonJS)